### DISCUSSION TITLE
when I run  cargo leptos watch  I get an  error

### DISCUSSION THREAD

**Discussion by GitGreenHand** [wasm-bindgen/wasm-bindgen#4240] [Q&A]
➜  moonbound git:(main) ✗ rustup target add wasm32-unknown-unknown
info: downloading component 'rust-std' for 'wasm32-unknown-unknown'
info: installing component 'rust-std' for 'wasm32-unknown-unknown'
➜  moonbound git:(main) ✗ cargo leptos watch                      
   Compiling cfg-if v1.0.0
   Compiling once_cell v1.20.2
   Compiling memchr v2.7.4
   Compiling futures-core v0.3.31
   Compiling futures-sink v0.3.31
   Compiling pin-project-lite v0.2.14
   Compiling serde v1.0.210
   Compiling thiserror v1.0.64
   Compiling slab v0.4.9
   Compiling futures-task v0.3.31
error[E0463]: can't find crate for `core`
  |
  = note: the `wasm32-unknown-unknown` target may not be installed
  = help: consider downloading the target with `rustup target add wasm32-unknown-unknown`

error[E0463]: can't find crate for `std`
  |
  = note: the `wasm32-unknown-unknown` target may not be installed
  = help: consider downloading the target with `rustup target add wasm32-unknown-unknown`

➜  moonbound git:(main) ✗ rustup target list --installed          
aarch64-apple-darwin
wasm32-unknown-unknown

**Comment by daxpedda** [comment]
You might be better served asking at Leptos.

My best guess is that it might be trying to use a different toolchain, not your default one.
